README.md

```diff
@@ -34,6 +34,7 @@ Note that this extension requires the [Integrated Terminal plugin](https://plugi
 | ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Start                        | Starts dev mode.                                                                                                                                                                                                                                                                                                             |
 | Start…​                      | Starts dev mode with custom parameters. Supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#additional-parameters) and the [libertyDev task of the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters). |
+| Start in container (tech preview)                         | Starts dev mode in a container. The `liberty-maven-plugin` must be version `3.3-M1` or higher. The `liberty-gradle-plugin` must be version `3.1-M1` or higher.
```

Comment:
Describe it as something like `Starts dev mode with the server in a container.` - since dev mode itself is not in the container